apps/front-end/back-office/src/app/components/util/file-field/file-field.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, ElementRef, HostListener } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+
+@Component({
+  selector: 'webonjour-file-field',
+  templateUrl: './file-field.component.html',
+  styleUrls: ['./file-field.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
```

Comment:
what is NG_VALUE_ACCESSOR

Reply:
https://medium.com/@majdasab/implementing-control-value-accessor-in-angular-1b89f2f84ebf